data/human/jobs.txt

```diff
@@ -4167,3 +4167,291 @@ mission "Republic Navy Advisory System [wildfire firefighters]"
 	on complete
 		payment 9000 85
 		dialog "You land at a makeshift landing site a few minutes flight from the blaze. The smoky haze that drifts into your ship when the firefighters open the hatch to depart makes your eyes and nose itch. As you watch your passengers heading toward a staging area nearby, you think about the inferno they will soon be voluntarily entering. A chirp from your communications system indicates your stipend of <payment> has been deposited, and you begin your atmospheric flight back to <planet>'s main spaceport."
+
+mission "Mineral Shortages (Silicon)"
+	name "Mineral shortages on <planet>"
```

Comment:
`Mineral shortages` might not be the best name here, because it doesn't tell you anything about the job. Maybe:

```suggestion
	name "Harvested silicon to <planet>"
```

Could even remove "harvested" from that, and just do them along the lines of `Silicon to <planet>`